CHANGELOG.md

```diff
@@ -1,6 +1,14 @@
 Changelog
 =========
 
+## 3.0.1 (2022-03-24)
+
+### Bug fixes
+
+* Ensure RemoveProjectRoots works for assemblies that are run on a OS that differs from their build OS.
+  | [sgtfrankieboy](https://github.com/sgtfrankieboy)
+  | [#154](https://github.com/bugsnag/bugsnag-dotnet/pull/154)
+
```

Comment:
It's a bit confusing seeing this entry here.  I guess we only just realised it was missing, as opposed to it being routine that we add changelogs for external contributions at a later date?

Reply:
yeah, I think this was just missing from `next` because it wasn't rebased after the previous release